### PR TITLE
Revise course-customization and README for dc-name and first push

### LINKED ISSUE #603: New forks don't build unless some change is pushed.
I forked a copy of the course, and tried viewing the github pages version, but it didn build.

I contacted support and they said:

"Since you had never pushed to that repository after forking it, a Pages build wasn't started."

Update the documentation to suggest pushing a change before browsing the pages.

### LINKED ISSUE #606: Update `docs/`
* add a note to the docs letting people know that if they are interested in using the "Data Carpentry" name that they should contact me

* add note to `contributing` about first `push` (#603)

### PR DESCRIPTION
**Pull request by brymz**
Fixes #606; Fixes #603

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,11 @@ The website is setup to be easy to run automatically through GitHub:
      `https://yourusername.github.io/semester-biology/`
 2. Update `# Setup` information in `_config.yml` in the main directory for
    proper site rendering.
+   * You must `push` this change to your repository to build and browse your 
+     forked version. 
+   * If you choose to keep the "Data Carpentry" name in your `title:`, please
```
Comment (ethanwhite) [inline README.md]: I don't think we need the title thing in the README.

File: docs/course/course-customization.md [hunk 2]
```diff
@@ -15,6 +15,11 @@ The website is setup to be easy to run automatically through [GitHub](http://git
 
 - Update `# Setup` information in `_config.yml` in the main directory for proper
   site rendering.
+  - You must `push` this change to your repository to build and browse your 
+    forked version. 
+  - If you choose to keep the "Data Carpentry" name in your `title:`, please
+    [contact us via email ({{ site.email }})](mailto:{{ site.email }})
```
Comment (ethanwhite) [inline docs/course/course-customization.md]: Missing `]`.

File: README.md (hunk 1 above)
Comment (brymz) [inline README.md]: Done.

File: docs/course/course-customization.md (hunk 2 above)
Comment (humberto-ortiz) [inline docs/course/course-customization.md]: I suggest moving the line "   - In a few minutes you should be able to see a copy of the site at:
     `https://yourusername.github.io/semester-biology/`" below the new line about pusing the change.

I understood it to mean "a few minutes after doing the fork you should be able to see".

**Pull request merged**